docs/why.qmd

```diff
@@ -2,24 +2,224 @@
 title: "Why Ibis?"
 ---
 
+Ibis defines a Python dataframe API that supports any query engine -- the
+frontend for any backend data platform. This includes 20+ backends today, most
+SQL-based.  This allows Ibis to have excellent performance -- as good as the
+backend it is connected to -- with a consistent user experience.
+
+## What is Ibis?
+
 Ibis is the portable Python dataframe library.
```

Comment:
My suggestion is to describe Ibis as a combination of two independent values and expand into concrete reasons why people should use Ibis over similar tools.
1. Lightweight Python dataframe for all your DS/analytics workload - we need examples to demonstrate why the API is more elegant, usable, performant than its competitors Pandas, Polars, PySpark, etc
2.Ability to for delayed execution on any modern compute engines - coupled with use case examples, maybe we should also get more (future) deep technical content to describe why Ibis is better.

Reply:
follow up for this: https://github.com/ibis-project/ibis/issues/8514